ssrando/ssrando
# replace pyqtdarktheme with fork and support python3.12

## What does this PR do?
Replace pyqtdarktheme with a fork (because the original project doesn't seem to be maintained)
## How do you test this changes?
Just open the GUI, make sure that the dark mode settings still work.

This does **NOT** need actual playthroughs to test.
